css/fonts.css

```diff
@@ -34,7 +34,6 @@
          url('PTF75F-webfont.woff') format('woff'),
          url('PTF75F-webfont.ttf') format('truetype'),
          url('PTF75F-webfont.svg#pt_serifbold') format('svg');
-    font-weight: normal;
+    font-weight: bold;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5576209)